packages/date-picker/src/DatePicker/DatePickerMenu/DatePickerMenu.types.ts

```diff
@@ -3,6 +3,14 @@ import { PopoverProps } from '@leafygreen-ui/popover';
 
 export type DatePickerMenuProps = Omit<
   PopoverProps,
-  'children' | 'renderMode' | 'usePortal'
+  | 'children'
+  | 'dismissMode'
+  | 'onToggle'
+  | 'popoverZIndex'
+  | 'portalClassName'
+  | 'portalContainer'
+  | 'portalRef'
+  | 'renderMode'
+  | 'scrollContainer'
```

Comment:
Could we simplify this as an exported type from Popover?

Reply:
I feel like the _ideal_ case would be for everything to expose the same props (for API consistency). For now while we're going for parity, this is fine